.travis.yml

```diff
@@ -82,6 +90,15 @@ jobs:
       script:
         - docker run cocotb powershell 'python setup.py install ; make test'
 
+    - stage: SimTests
+      python: 3.8
```

Comment:
That's not having any effect, if you want to use a different Python version, you need to do so in the Dockerfile.windows. 

I'd suggest removing the Windows part for now, Linux Python 3.8 testing is already an important step forward.

Reply:
Okay, Linux only it is now.